build.sbt

```diff
@@ -1,19 +1,19 @@
 import Dependencies._
 
-lazy val defaultSettings = Seq(
```

Comment:
@AlekseiLitkovetc made some changes on the structure of the build.sbt in his [PR](https://github.com/EpamLifeSciencesTeam/cmwl_pipeline/pull/4/files). Maybe we should not change the formatting of this file concurrently? :)

Reply:
I could take care about merging it, don't worry :)